src/ape_cache/_cli.py

```diff
@@ -0,0 +1,47 @@
+import click
+import pandas as pd
+
+from ape.utils import ManagerAccessMixin
+
+from . import models
+
+
+def get_engine():
+    return ManagerAccessMixin.query_manager.engines["cache"]
+
+
+@click.group(short_help="Query from caching database")
+def cli():
+    """
+    Command-line helper for managing query caching database.
+    """
+
+
+@cli.command(short_help="Initialize a new cache db")
+def init():
+    db_file = get_engine().database_file
+    if not db_file.is_file():
+        click.echo("initializing database")
+        models.Base.metadata.create_all(bind=get_engine().engine)
+        return click.echo("caching database initialized")
+    click.echo("caching database already exists!")
+
+
+# TODO: add migrate command
+
+
+@cli.command(short_help="Call and print SQL Statement to the cache db")
```

Comment:
```suggestion
@cli.command(short_help="Call and print SQL statement to the cache DB")
```

Reply:
I changed this to database rather than DB. I think it looks cleaner